### DISCUSSION TITLE
My Learnings and Resources from #100DaysOfKubernetes

### DISCUSSION THREAD

**Discussion by verma-kunal** [EddieHubCommunity/support#1717] [Show and tell]
I am following the **#100DaysOfKubernetes** challenge, which I got inspired to start from **[Anais](https://github.com/AnaisUrlichs)**

In this, each day I will be learning something new about Kubernetes and related topics, then posting my learnings on Twitter with some resources as well, to help folks out there who would like to join in as well !! 😃

In this discussion, you can follow through my journey so far and each day, through Twitter links as well as take help from the some resources I would be posting as well !!

List Of Links:
- My Journey  -> Github Repo ([Link](https://github.com/verma-kunal/100DaysOfKubernetesJourney))
- Resources -> 
Playlist on YT - ([Link](https://www.youtube.com/watch?v=W6J6UipVNGA&list=PLWnens-FYbIpUpmiiNYfkqTZQUYppGMFV&index=1))
K8s Documentation - ([Link](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/))
Book - [100DaysOfKuberntes](https://100daysofkubernetes.io/)

You can find further resources in my Daily Tweets as well (In the GitHub **Journey repo** or on [Twitter](https://twitter.com/kverma_twt))

**Comment by verma-kunal** [comment]
### Day-8
**List of Basic Docker Commands to keep them handy** : https://hackmd.io/@akd1OSHeTmmTou_vRDXfxQ/rJmQVrFYd

**Comment by eddiejaoude** [comment]
Awesome 👍
